src/services/edge-application-rules-engine-services/delete-rules-engine-service.js

```diff
@@ -0,0 +1,57 @@
+import { AxiosHttpClientAdapter } from '../axios/AxiosHttpClientAdapter'
+import { makeEdgeApplicationBaseUrl } from '../edge-application-services/make-edge-application-base-url'
+import * as Errors from '@/services/axios/errors'
+
+export const deleteRulesEngineService = async ({ ruleId, phase, edgeApplicationId }) => {
+  let httpResponse = await AxiosHttpClientAdapter.request({
+    url: `${makeEdgeApplicationBaseUrl()}/${edgeApplicationId}/rules_engine/${phase}/rules/${ruleId}`,
+    method: 'DELETE'
+  })
+
+  return parseHttpResponse(httpResponse)
+}
+/**
+ * @param {Object} errorSchema - The error schema.
+ * @param {string} key - The error key of error schema.
+ * @returns {string|undefined} The result message based on the status code.
+ */
+const extractErrorKey = (errorSchema, key) => {
+  return `${errorSchema[key]}`
+}
+
+/**
+ * @param {Object} httpResponse - The HTTP response object.
+ * @param {Object} httpResponse.body - The response body.
```

Comment:
@param {Object} httpResponse.body - The response body. 
mudar para
@param {Array<Object>} httpResponse.body - The response body.